README.md

```diff
@@ -27,13 +25,47 @@ cd lkvs
 make
 ```
 
+## Alternativly compile the whole project with Docker
+
+```
+make docker_image
+make docker_make
+```
+
 ## Compile a single test case
 
 ```
 cd lkvs/<test>
 make
 ```
 
+# Run tests
+
+There're two ways to run the tests.
+
+## Binary/Shell directly
+
+Normally, there're one or more executable binaries or scirpts in a sub-project, once it is compiled successfully. The easiest way is to run them directly.
+
+## Case runner
+
+**runtests** is a straightforward test runner that can execute a single or a set of test cases and redirect the log.
+
+There are 2 ways to pass test to **runtests**:
+  1. `-c`: Pass test cmdline.
+  2. `-f`: Record the cmdline in a `tests` file, then pass the file to the runtests, which can contain multiple tests.
```

Comment:
"
`

> 2. `-f`: Record the cmdline in a `tests|tests-client|tests-server` file, all these tests files are in each component folder, please fill the tests files after -f.

`

Other thing is good to me.

Thanks!

Reply:
`tests*` should be better, cause we only need the file name starts with **tests**. Asterisk can be replaced by any resonable strings.

Reply:
Yes, tests* is better for me.

> -f: Record the cmdline in a tests* file, all these tests* file are in each component folder, please fill the tests* files after -f.

Thanks!